src/table/cells/handle-key-press.js

```diff
@@ -54,17 +54,42 @@ export const preventDefaultBehavior = (evt) => {
   evt.preventDefault();
 };
 
-const handleKeyPress = (evt, rootElement, rowIndex, colIndex, cell, selState, selDispatch) => {
+export const moveFocus = (evt, rootElement, rowIndex, colIndex, selState) => {
+  preventDefaultBehavior(evt);
+  removeCurrentFocus(evt);
+  const rowAndColumnCount = getRowAndColumnCount(rootElement);
+  const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex, selState);
+  moveToNextFocus(rowAndColumnCount.rowElements, nextRow, nextCol);
+};
+
+export const headHandleKeyPress = (evt, rootElement, rowIndex, colIndex, changeSortOrder, layout, isDim) => {
   switch (evt.key) {
     case 'ArrowUp':
     case 'ArrowDown':
     case 'ArrowRight':
     case 'ArrowLeft': {
+      moveFocus(evt, rootElement, rowIndex, colIndex);
+      break;
+    }
+    // Space bar / Enter: update the sorting
+    case ' ':
+    case 'Enter': {
       preventDefaultBehavior(evt);
-      removeCurrentFocus(evt);
-      const rowAndColumnCount = getRowAndColumnCount(rootElement);
-      const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex, selState);
-      moveToNextFocus(rowAndColumnCount.rowElements, nextRow, nextCol);
+      changeSortOrder(layout, isDim, colIndex);
+      break;
+    }
+    default:
+      break;
+  }
+};
+
+const handleKeyPress = (evt, rootElement, rowIndex, colIndex, cell, selState, selDispatch) => {
```

Comment:
I'd say this is not the default function anymore and should be renamed. Since the other one is `headHandleKeyPress` I suggest calling this `bodyHandleKeyPress`

Reply:
Fixed. Thanks